notebooks/data-sources/oc-github-repo/github_PR_log_models.ipynb

```diff
@@ -0,0 +1,1038 @@
+{
```

Comment:
Do you think the [`notebooks/time-to-merge-prediction`](https://github.com/aicoe-aiops/ocp-ci-analysis/tree/697b1f07deabce5bb99fbd498ac11fba14d0e63a/notebooks/time-to-merge-prediction) folder would be a better place to have this notebook? IIUC the `notebooks/data-sources/oc-github-repo` folder is more for exploratory work.